lib/modules/datasource/deb/readme.md

```diff
@@ -55,3 +58,50 @@ RUN apt-get update && \
 ```
 
 When the apt package for `gcc` is updated, Renovate updates the environment variable.
+
+```json title="Overwrite deb registryUrl via packageRule"
+{
+  "packageRules": [
+    {
+      "matchDatasources": ["deb"],
+      "matchPackageNames": ["gcc-11"],
+      "registryUrls": [
+        "https://deb.debian.org/debian?suite=stable&components=main,contrib,non-free&binaryArch=amd64"
```

Comment:
Should we add an info that multiple registiryUrls are supported and the merge Strategy is used?

Reply:
The first fact is true for all datasources the second makes more sense to add to the content table automatically. 
Like we do already with e.g. customRegistry